mobile/examples/object_detections/iOS/ORTObjectDetection/ModelHandler.swift

```diff
@@ -0,0 +1,463 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
why both MIT and Apache licenses?

Reply:
Because some of the code are from files licensed under Apache licenses. You suggest only add Apache licenses?

Reply:
oh, I see - was it copied from some other project and modified?
we've also done something like this:
https://github.com/microsoft/onnxruntime/blob/1d8edd0b5bcf3c3172cc10ae60662073e8f8f748/include/onnxruntime/core/common/common.h#L1-L16
I'm not sure what the proper way to handle it is though